### PR TITLE
issue#23 and Issue#5

### PR DESCRIPTION
**Pull request by chimericalidea**
Renaming all the stuff as in #23. And added new image with corrected names. 
Also addressed issue #5.

### CODE REVIEW

File: pages/architecture.md [hunk 1]
```diff
@@ -18,25 +18,29 @@ together.
...
+There are two primary transports: HTTP and Kafka. Scribe can also be used.
+See [Span Receivers]({{ site.github.url }}/pages/span_receivers) for more information.
 
+![Architecture overview]({{ site.github.url }}/graphs/architecture/graph.png)
 There are 4 components that make up Zipkin:
+
 * collector
-* storage
-* search
+* database
```
Comment (abesto) [inline pages/architecture.md]: Storage is still storage. Database is the component that storage talks to, and is not strictly part of Zipkin.

File: pages/architecture.md [hunk 2]
```diff
@@ -18,25 +18,29 @@ together.
...
 
+
+
+
 ### Zipkin Collector
 
 Once the trace data arrives at the Zipkin collector daemon, it is validated, stored, and indexed for lookups by the Zipkin collector.
 
-### Storage
+### Database
```
Comment (abesto) [inline pages/architecture.md]: Same as above.

File: pages/instrumenting.md [hunk 3]
```diff
@@ -144,10 +140,9 @@ Ids are encoded as [hex strings](https://github.com/twitter/finagle/blob/master/
 * X-B3-SpanId: 64 encoded bits
 * X-B3-ParentSpanId: 64 encoded bits
 * X-B3-Sampled: Boolean (either "1" or "0")
-* X-B3-Flags: a Long
+* X-B3-Flags: a Long string
```
Comment (abesto) [inline pages/instrumenting.md]: `Long` is a type in itself (usually means a 64bit integer)

**Comment by abesto** [comment]
LGTM. @adriancole, feel like taking a look?

File: pages/architecture.md [hunk 4]
```diff
@@ -18,20 +18,24 @@ together.
...
 Transport
 ---------
 
-Spans sent by the instrumented library must be transported from the services being traced to Zipkin collectors.
-There are two primary transports: Scribe and Kafka. Scribe is shown in the figure. See [Span Receivers]({{ site.github.url }}/pages/span_receivers) for more information.
+Tracing data sent by the instrumented library must be transported from the services being traced to Zipkin collectors.
+There are two primary transports: HTTP and Kafka. Scribe can also be used.
+
+![Architecture overview]({{ site.github.url }}/graphs/architecture/graph.png)
+There are five components that make up Zipkin:
```
Comment (adriancole) [inline pages/architecture.md]: this will scare people as there's already a stereotype zipkin is hard to install. Maybe change this sentence to..

While zipkin can run in a single process, there are N components that make up Zipkin architecture

File: pages/architecture.md [hunk 5]
```diff
@@ -18,20 +18,24 @@ together.
...
 
-There are 4 components that make up Zipkin:
 * collector
 * storage
-* search
+* query
```
Comment (adriancole) [inline pages/architecture.md]: let's not separate query from API, as it it looks like there's more parts than there are

File: pages/architecture.md [hunk 6]
```diff
@@ -46,6 +50,10 @@ component pluggable. In addition to Cassandra, we support ElasticSearch and MySQ
 
 Once the data is stored and indexed, we need a way to extract it. The query daemon provides a simple JSON API for finding and retrieving traces. The primary consumer of this API is the Web UI.
 
+### API
+
+This is the Zipkin component which provides an HTTP API to the UI for accessing tracing data.
```
Comment (adriancole) [inline pages/architecture.md]: yep and can remove the Query section, if exists

File: pages/instrumenting.md [hunk 7]
```diff
@@ -28,33 +28,24 @@ First, there are a core set of structures that we need:
...
-* **sr** - Server Receive: The server has received the request and will start
-  processing it. The difference between this and `cs` will be combination of
-  network latency and clock jitter.
-* **ss** - Server Send: The server has completed processing and has sent the
-  request back to the client. The difference between this and `ss` will be the
-  amount of time it took the server to process the request.
-* **cr** - Client Receiver: The client has received the response from the server.
-  This sets the end of the span. The RPC is considered complete when this
-  annotation is recorded.
+* **cs** - Client Start. The client has made the request. This sets the beginning of the span.
```
Comment (adriancole) [inline pages/instrumenting.md]: can you summarize if anything changed here except formatting?

**Comment by adriancole** [comment]
just a couple small notes, then :ship:

**Comment by adriancole** [comment]
ping :)

**Comment by adriancole** [comment]
note: this or similar needs to go in soon, since we've converged the codebase recently.

**Comment by adriancole** [comment]
closing this as it has had no activity in 4 months
